runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -64,18 +76,39 @@
 	private final JLabel overallGpLabel = new JLabel();
 	private final JLabel overallIcon = new JLabel();
 
+	private final JPanel actionsContainer = new JPanel();
```

Comment:
Are you sure all those need to be instance fields? And not created only where they are used?